universal-application-tool-0.0.1/app/views/questiontypes/ApplicantQuestionRenderer.java

```diff
@@ -1,8 +1,67 @@
 package views.questiontypes;
 
+import static com.google.common.base.Preconditions.checkNotNull;
+import static j2html.TagCreator.div;
+
+import j2html.tags.ContainerTag;
 import j2html.tags.Tag;
+import play.i18n.Messages;
+import services.applicant.question.ApplicantQuestion;
+import views.BaseHtmlView;
+import views.style.ApplicantStyles;
+import views.style.ReferenceClasses;
+import views.style.Styles;
+
+public abstract class ApplicantQuestionRenderer {
```

Comment:
praise: love that this is an abstract class!

Reply:
wohoo! 😃